styles.css

```diff
@@ -86,8 +101,45 @@ body {
   margin-right: auto;
 }
 
+nav {
+  width: 100%;
+}
+
 .desc-nav {
   display: none;
+  position: absolute;
+  width: 100%;
+  margin-top: 50px;
+  margin-left: 28px;
+}
```

Comment:
The menu's style doesn't match the Figma design.

<img width="228" alt="Screen Shot 2022-07-20 at 18 25 16" src="https://user-images.githubusercontent.com/82426682/180099248-35273e46-c1f0-410d-a365-192d02e28e0d.png">

<img width="277" alt="Screen Shot 2022-07-20 at 18 25 39" src="https://user-images.githubusercontent.com/82426682/180099282-d5cad0ec-a8de-4161-875a-97e61dd41462.png">

Please make sure you add opacity to the background, make sure you fix the content's font and font size and make sure you fix the `X` icon's size and positioning.

Additionally, make sure that, whilst the menu is open, that is all the user can see even if they scroll up and down the page.

<img width="226" alt="Screen Shot 2022-07-20 at 18 27 37" src="https://user-images.githubusercontent.com/82426682/180099496-1d7b111e-9f5b-4c9f-879d-70cd881222b3.png">

Reply:
Hi @fullstop125, while you did great implementing some of the features asked by the previous reviewer, your design for the nav menu still does not match the Figma template. Consider fixing it as requested by the previous reviewer.
This is how your application is looking now. ⬇️
![image](https://user-images.githubusercontent.com/33268927/180181893-4a946696-5749-41b5-84e9-5c8050545b41.png)

Reply:
@muneebulrehman hello, it seems there is an issue on your end ...on my local computer, it loads just fine ...here is a screenshot  and a link to the loom recording : https://www.loom.com/share/f56a6f490d5e4d33a3da799e94a8bde8


![image](https://user-images.githubusercontent.com/35464104/180183659-f0214453-a4fc-4450-bc97-03d01c323d70.png)